.gitignore

```diff
@@ -21,4 +21,5 @@ target
 
 build
 dist
+output.json
```

Comment:
`.gitignore` is used to ignore files that are widely considered either hidden or temporary. It seems safer not to include `output.json` in `.gitignore`.